android/ui/src/main/java/com/trafi/ui/theme/CornerRadius.kt

```diff
@@ -18,10 +18,11 @@ object CornerRadius {
 }
 
 @Stable
-internal val Dp.isRound: Boolean
+val Dp.isRound: Boolean
     get() = this == CornerRadius.round
 
 @Immutable
 data class MaasCornerRadius(
-    val buttonRadius: Dp = CornerRadiusScale.Default.ButtonRadius
+    val buttonRadius: Dp = CornerRadiusScale.Default.ButtonRadius,
+    val nearbyTransitFilterItemRadius: Dp = CornerRadiusScale.Default.NearbyTransitFilterItemRadius
```

Comment:
Why should this be part of our public theme API? Unless we have a good reason, I'd suggest hardcoding it in `NearbyTransitFilterItemConstants` for now.

```kotlin
val cornerRadius = CornerRadiusScale.round
```

Reply:
Agreed!